waterdrop.gemspec

```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'delivery_boy', '~> 0.2'
   spec.add_dependency 'dry-configurable', '~> 0.8'
   spec.add_dependency 'dry-monitor', '~> 0.3'
-  spec.add_dependency 'dry-validation', '~> 0.11'
-  spec.add_dependency 'ruby-kafka', '>= 0.7.1'
+  spec.add_dependency 'dry-validation', '~> 1.0.0.rc1'
```

Comment:
I *think* if you set it to `~> 1.0.0` already it will work with RCs too.

Reply:
Nope:
```Bundler could not find compatible versions for gem "dry-validation":
  In Gemfile:
    dry-validation

    waterdrop was resolved to 1.3.0, which depends on
      dry-validation (~> 1.0.0)

Could not find gem 'dry-validation (~> 1.0.0)', which is required by gem 'waterdrop', in any of the relevant sources:
  https://github.com/dry-rb/dry-validation.git (at master@d748280)

```